tests/kernels/test_cutlass.py

```diff
@@ -221,6 +224,135 @@ def test_cutlass_int8_gemm_m_sweep(per_act_token: bool, per_out_ch: bool,
                                      use_bias)
 
 
+def naive_mm(a: torch.Tensor, b: torch.Tensor) -> torch.Tensor:
```

Comment:
doesn't look like this is used? or am I missing something?

Reply:
Yep I fixed it, good catch